### PR TITLE
feat(infra): scaffolding for different types of elements

### PR DESCRIPTION
**Pull request by forforeach**
Add an ability to generate different types of elements, including full structure and story registration.
Currently available scaffolding types:
- **Components** - will generate component structure and register it to stories. Also, prefixes the given name with `Fbx`
  >Usage: `npx hygen component new --name ComponentName`

- **Directives** - will generate directive structure and register it to stories. Also, prefixes the given name with `Fbx`
  >Usage: `npx hygen directive new --name DirectiveName`

**cc**: @Fundbox/vue-committee

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,16 @@
-# Fundbox UI Components
+# Fundbox UI
 
+
+## Development
+
+### Scaffolding
+In order to improve productivity you can generate new elements' structure with CLI commands. We use [hygen](http://www.hygen.io/) for this porpouse. There are couple of ways to use `hygen`:
```
Comment (nlitwin) [inline README.md]:
```suggestion
In order to improve productivity you can generate new elements' structure with CLI commands. We use [hygen](http://www.hygen.io/) for this purpose. There are couple of ways to use `hygen`:
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,16 @@
...
+1. `npm i -g hygen` - which installs `hygen` globally
+2. `npx hygen ...` - which runs hygen remotly
```
Comment (nlitwin) [inline README.md]:
```suggestion
2. `npx hygen ...` - which runs `hygen` remotely
```

File: _templates/component/new/component.stories.ejs.t [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+to: components/Fbx<%= name %>/Fbx<%= name %>.stories.js
+---
+
```
Comment (nlitwin) [inline _templates/component/new/component.stories.ejs.t]: An extra line at the top of the file is created
```suggestion
```

File: _templates/component/new/inject_component.ejs.t [hunk 4]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (nlitwin) [inline _templates/component/new/inject_component.ejs.t]: 👌🏼You're a gentleman and a scholar

File: _templates/component/new/component.stories.ejs.t (hunk 3 above)
Comment (forforeach) [reply _templates/component/new/component.stories.ejs.t]: Removed

File: _templates/component/new/inject_component.ejs.t (hunk 4 above)
Comment (forforeach) [reply _templates/component/new/inject_component.ejs.t]: Thank you 🙂

**Pull request merged**